src/Behat/MinkExtension/ServiceContainer/Driver/AppiumFactory.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+
+namespace Behat\MinkExtension\ServiceContainer\Driver;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+
+class AppiumFactory extends Selenium2Factory
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDriverName()
+    {
+        return 'appium';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(ArrayNodeDefinition $builder)
+    {
+        $builder
+            ->children()
+                ->scalarNode('browser')->defaultValue('remote')->end()
+                ->append($this->getCapabilitiesNode())
+                ->scalarNode('appium_host')->defaultValue(getenv('APPIUM_HOST'))->end()
+                ->scalarNode('appium_port')->defaultValue(getenv('APPIUM_PORT'))->end()
```

Comment:
-1 for using environment variables as default values. It would mean that the config is broken in case the default values are missing. Use the default host and port as default values instead.